rlbot_gui/gui/css/style.css

```diff
@@ -1,7 +1,6 @@
 @import url('https://fonts.googleapis.com/css?family=Exo+2:300,300i,500,500i');
 
 :root {
-  --main-color: #fff;
```

Comment:
I have tried to avoid "core" rlbot_gui things to avoid unintended impact if we merge the projects. I added a `story.css` that is already included, maybe we can leverage that?

Reply:
Thanks for being cautious! I maintain the core stuff regularly and I happened to notice that style was superfluous and probably left over from when we were using material ui.